lua/entities/gmod_tardis/modules/teleport/sh_tp_main.lua

```diff
@@ -307,9 +307,7 @@ if SERVER then
             and not self:GetData("redecorate")
             and not self:GetData("redecorate_parent")
         then
-            self:Timer("VortexChangePositionTime", 3, function()
-                self:ChangePosition(self:GetRandomLocation(false), self:GetAngles(), false)
-            end)
+            self:ChangePosition(self:GetRandomLocation(false), self:GetAngles(), false)
```

Comment:
What issues were there with the timer? I'd much rather fix those if possible it's a pretty jarring jump as soon as you finish dematting. We maybe just need to ensure that we are still in the correct state (dematted, in vortex, not matting, tardis is valid etc)

Reply:
The vortex just glitches out
When it is moved and the TARDIS is moved, the vortex is invisible until you start moving your camera around
It happens often, and looks horrible

Reply:
I've seen this happen before with the standard vortex as well, but I agree I have seen this happening as well. I think given the problem that it currently has this is a better solution for now, but I'd like to log it as an issue to be investigated but I have a suspicion that it's going to be down to limitations in rendering.